commands.go

```diff
@@ -125,6 +125,12 @@ func (cmd Command) Run(label string, cmdline ...string) error {
 		exe.Env = append(os.Environ(), cmd.extraEnv...)
 	}
 
+	// Disable services start/stop for commands running in chroot
+	if cmd.ChrootMethod != CHROOT_METHOD_NONE {
+		ServicePolicyHelper(cmd.Chroot, DenyServices)
```

Comment:
Given the only usage of the Helper is to deny things; seems more elegant to follow the pattern of the qemuHelper; iotw instatiate an object to Deny services and defer a Cleanup function on it. Or if you prefer do something like:
```go
s := ServicePolicyHelper(cmd.chroot)
s.Deny()
defer s.Allow()
```
In your current code your one function is just a dispatch for two different functions essentially :smile: